.github/workflows/deploy_to_github_io.yml

```diff
@@ -27,7 +27,9 @@ jobs:
         shell: bash -l {0}
         run: |
           cd hyp3-docs
-          mkdocs gh-deploy
+          # Remove conflicting CNAME for project site
+          rm docs/CNAME
+          mkdocs gh-deploy --force
```

Comment:
Can't conditionally remove w/out installing a plugin which is a little heavyweight for just this file:
https://github.com/mkdocs/mkdocs/issues/1500